app/assets/javascripts/miq_ujs_bindings.js

```diff
@@ -83,26 +80,31 @@ $(document).ready(function () {
         if (el.attr('data-miq_sparkle_off')) {
           options.complete = true;
         }
-        miqJqueryRequest(url, options);
+        $.when(miqJqueryRequest(url, options)).done(function() {
+          if (parms.auto_refresh === true) {
+            dialogFieldRefresh.triggerAutoRefresh(parms.field_id, parms.trigger);
+          }
+        });
       });
     } else {
       $(this).off(); // Use jQuery to turn off observe_field, prevents multi ajax transactions
       var el = $(this);
       el.observe_field(interval, function () {
-        if (parms.auto_refresh === true) {
-          dialogFieldRefresh.triggerAutoRefresh(parms.field_id, parms.trigger);
-        }
-
+        var attemptAutoRefreshTrigger = function() {
```

Comment:
I'm thinking there must not be a way to share this function - bummer it has to be defined twice.